### DISCUSSION TITLE
Push to LFS without Git commit

### DISCUSSION THREAD

**Discussion by sebastianrath** [git-lfs/git-lfs#4558] [Q&A]
Hello, does anyone know if there is a way to push a file object to LFS without the object being part of a commit? In my specific use case I have a few files that are part of a project but not really of the repo and I need a place to temporarily park them. Any hint is highly appreciated!

**Comment by bk2204** [comment]
Hey,

Yes, this is possible with `git lfs push --object-id`.  Do be aware that some server-side implementations could prune objects unrelated to the repository; I'm not aware of any that presently do this, but it is a possible implementation choice.  If they did that, then you could end up with data missing.

**Reply by sebastianrath** [reply]
Thank you, it works! That helped a lot!
